sentry_flink/sentry_flink/flink/flink_translator.py

```diff
@@ -78,6 +68,22 @@ def translate_to_flink_type(type: Any) -> TypeInformation:
         return fn(arg1(), arg2())
 
 
+def translate_custom_type(type: Any) -> TypeInformation:
+    """
+    For Python types which use Pickle for serialization/deserialization
+    and do not have a matching standard Flink Type. e.g. custom-defined
+    classes. Also handles Generator type.
+    """
+
+    assert not is_standard_type(type)
+
+    if isinstance(type, GeneratorType):
+        return translate_to_flink_type(get_args(type)[0])
+
+    else:
+        return Types.PICKLED_BYTE_ARRAY()
```

Comment:
Does this mean Flink pickles objects to send them around between workers?

Reply:
For types that do not have a 1:1 equivalent in Flink's type system, you can use this type. It can be used for any type which uses Pickle for (de)serialization https://github.com/apache/flink/blob/3c31e646a8efc13943b33d012a70d748cf11cdbc/flink-python/pyflink/common/typeinfo.py#L892-L898